src/Conference.tsx

```diff
@@ -442,76 +448,76 @@ const Conference: React.FC = (props) => {
   }
 
   return (
-      <div
-          className={immersiveMode ? classes.conferenceNoCursor : classes.conference}
-          onMouseMove={() => setImmersiveMode(false)}
-      >
-        {settingsModalOn && <Settings toggleSettings={toggleSettings} onSubmit={handleSettingsSubmit} />}
-        <SessionInfo
-            immersiveMode={immersiveMode}
-            userAgent={userAgent}
-            conferenceId={conferenceId}
-            conferenceCode={conferenceCode}
-            conferencePhoneNumber={phoneNumber}
-            participantId={participantId}
-            localStream={localStream}
-            remoteStreams={Object.values(remoteStreams)}
-            error={error}
+    <div
+      className={immersiveMode ? classes.conferenceNoCursor : classes.conference}
+      onMouseMove={() => setImmersiveMode(false)}
+    >
+      {settingsModalOn && <Settings bandwidthRtc={bandwidthRtc} toggleSettings={toggleSettings} onSubmit={handleSettingsSubmit} />}
```

Comment:
I think we should process the TURN server settings in the handleSettingsSubmit callback rather than supplying BandwidthRTC to the Settings component. All you need to do is add another argument to the callback function

Reply:
I change the code to fix this problem